.eslintrc.json

```diff
@@ -23,7 +23,6 @@
     "react/display-name": "off",
     "react/no-unescaped-entities": "warn",
     "react-hooks/rules-of-hooks": "error",
-    "react-hooks/exhaustive-deps": "error",
```

Comment:
this is an important react/eslint rule that shouldn't be taken off imho.

Reply:
"react-hooks/exhaustive-deps" was set twice in that file, once with "off" and once with "error". tbh sometimes i found it annoying, but it might be better to not completely ditch it. maybe make it into a warning instead?

Reply:
sure warning should be fine. 
we should ensure to fix these lint warnings before merging though, because this lint rule helps us from some unwanted bugs that may prop up with not having the correct dependencies or having unwanted ones.

Reply:
@vidvidvid https://dev.to/wkrueger/never-ignore-the-exhaustive-deps-rule-2ap8

Reply:
let's put the error back in then :)